website_portal_purchase/__init__.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Antiun Ingeniería S.L. - Sergio Teruel
+# (c) 2015 Antiun Ingeniería S.L. - Carlos Dauden
```

Comment:
The OCA CLA actually lets you maintain all copyright, it just hands over some licensing and packaging rights to OCA. But yeah that nuance is kind of out of scope here anyways.

You're right about the implicit copyright too, which boils back down to my suggestion that the real copyright is the Git blame log.

So I've slept on this, as well as dove into a bunch of PRs looking at examples. Don't worry about changing the `(c)`, it's too proliferated through everything else. I feel like if I blocked here I would need to block everywhere else, and I simply don't have the energy for something that isn't even being used correctly.